[2022-07-27 15:08:52,196] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag.training_model_C scheduled__2022-07-26T00:00:00+00:00 [queued]>
[2022-07-27 15:08:52,219] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag.training_model_C scheduled__2022-07-26T00:00:00+00:00 [queued]>
[2022-07-27 15:08:52,219] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 15:08:52,219] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2022-07-27 15:08:52,219] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 15:08:52,256] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): training_model_C> on 2022-07-26 00:00:00+00:00
[2022-07-27 15:08:52,266] {standard_task_runner.py:52} INFO - Started process 1724 to run task
[2022-07-27 15:08:52,277] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag', 'training_model_C', 'scheduled__2022-07-26T00:00:00+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpkz4bbeyk', '--error-file', '/tmp/tmpvwvxysbz']
[2022-07-27 15:08:52,278] {standard_task_runner.py:80} INFO - Job 68: Subtask training_model_C
[2022-07-27 15:08:52,455] {task_command.py:371} INFO - Running <TaskInstance: dag.training_model_C scheduled__2022-07-26T00:00:00+00:00 [running]> on host c054a673cb8f
[2022-07-27 15:08:52,670] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-07-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-26T00:00:00+00:00
[2022-07-27 15:08:52,673] {python.py:173} INFO - Done. Returned value was: 8
[2022-07-27 15:08:52,826] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag, task_id=training_model_C, execution_date=20220726T000000, start_date=20220727T150852, end_date=20220727T150852
[2022-07-27 15:08:52,894] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-27 15:08:52,984] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: dag.training_model_b scheduled__2022-07-26T00:00:00+00:00 [removed]>' for dag 'dag'. Marking it as removed.
[2022-07-27 15:08:52,986] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: dag.training_model_a scheduled__2022-07-26T00:00:00+00:00 [removed]>' for dag 'dag'. Marking it as removed.
[2022-07-27 15:08:52,987] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: dag.training_model_c scheduled__2022-07-26T00:00:00+00:00 [removed]>' for dag 'dag'. Marking it as removed.
[2022-07-27 15:08:53,007] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
